### PR TITLE
Exercises

### PR DESCRIPTION
**Pull request by nikos-terzo**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,6 @@ move/lesson/build
 setup/*.env
 .publish.res.json
 .vercel
+
+# pnpm
+pnpm-lock.yaml
```
Comment (teohaik) [inline .gitignore]: pnpm lock (like package-lock) should not be ignored.

https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore

File: move/lesson/sources/ride.move [hunk 2]
```diff
@@ -44,6 +44,23 @@ module suitaxi::ride {
         initial_location: String
     }
 
+    struct RideAcceptedEvent has drop, copy {
```
Comment (teohaik) [inline move/lesson/sources/ride.move:47]: Make sure that the flow in Move file is:

std uses
sui uses
consts
structs 
events
init()
business functions
helper functions

**Review by teohaik — CHANGES_REQUESTED** [review]

File: move/lesson/sources/ride.move (hunk 2 above)
Comment (nikos-terzo) [reply move/lesson/sources/ride.move:47]: I see that the flow is as described

File: .gitignore (hunk 1 above)
Comment (nikos-terzo) [reply .gitignore]: Updated .gitignore (removed lines, added pnpm-lock.yaml) in [fbcab6b](https://github.com/teohaik/move-training-for-solutions-eng/pull/4/commits/fbcab6b80671e18d8b4f154edcd67e17ee15b9ed)

**Pull request closed without merging**